behat.yml

```diff
@@ -0,0 +1,34 @@
+# Run asset-admin behat tests with this command (installed with silverstripe/installer)
+# Note that asset-admin behat tests require CMS module
+# ========================================================================= #
+# vendor/bin/selenium-server-standalone -Dwebdriver.firefox.bin="/Applications/Firefox31.app/Contents/MacOS/firefox-bin"
+# vendor/bin/serve --bootstrap-file vendor/silverstripe/cms/tests/behat/serve-bootstrap.php
+# vendor/bin/behat @asset-admin
+# ========================================================================= #
+default:
+  suites:
+    campaign-admin:
+      paths:
+        - %paths.modules.campaign-admin%/tests/behat/features
+      contexts:
+        - SilverStripe\CampaignAdmin\Tests\Behat\Context\FeatureContext
+        - SilverStripe\Framework\Tests\Behaviour\CmsFormsContext
+        - SilverStripe\Framework\Tests\Behaviour\CmsUiContext
+        - SilverStripe\BehatExtension\Context\BasicContext
+        - SilverStripe\BehatExtension\Context\LoginContext
+        -
+          SilverStripe\CampaignAdmin\Tests\Behat\Context\FixtureContext:
+            - %paths.modules.campaign-admin%/tests/behat/files/
+        -
+          SilverStripe\Framework\Tests\Behaviour\ConfigContext:
+            - %paths.modules.campaign-admin%/tests/behat/files/
+  extensions:
+    SilverStripe\BehatExtension\Extension:
+      bootstrap_file: vendor/silverstripe/cms/tests/behat/serve-bootstrap.php
```

Comment:
Since we're using cms here let's add`cms` to the require-dev (same as asset-admin does). This is sufficient for dependencies only required during testing.